mypy_bootstrap.ini

```diff
@@ -15,3 +18,4 @@ platform = mypyc
 # needs py2 compatibility
 [mypy-mypy.test.testextensions]
 disallow_untyped_defs = False
+check_untyped_defs = False
```

Comment:
Wait, why do we need these tests here? I though `mypy_extensions` were moved to a separate repo, but it looks like the tests were left behind @msullivan will you have time to move tests and setup Travis in the new repo?

Reply:
Yeah this is because I had forgotten about this and assumed we just didn't have any tests for it. I can handle this.